src/main/java/com/github/manolo8/darkbot/extensions/features/FeatureRegistry.java

```diff
@@ -65,12 +73,32 @@ public void beforeLoad() {
     @Override
     public void afterLoad() {
         pluginHandler.LOADED_PLUGINS.forEach(pl ->
-                Arrays.stream(pl.getDefinition().features)
+                getFeaturesList(pl)
```

Comment:
I'd make this an either-or, so you either use `pl.getDefinition().features`, or, if it's null (the author doesn't define them in the list) then you search for the classes annotated with `@Feature` instead
That way the author may define them in either the "old way" (for backwards compat reasons), or simply clear the list, and then we'll search for all features ourselves